src/components/Header.jsx

```diff
@@ -1,18 +1,37 @@
-import { BsPersonFill } from 'react-icons/bs';
+import { useEffect, useState } from 'react';
+
+export default function Header() {
+  const [name, setName] = useState('');
+  const [profileImg, setProfileImg] = useState('');
+
+  const requestData = () => {
+    setName('Lidia Moroz');
+    setProfileImg('./img/profile_1.png');
+  };
+
+  useEffect(() => {
+    requestData();
+  }, []);
+
+  const styles = {
+    backgroundImage: `url(${profileImg})`,
+  };
 
-function Header() {
   return (
-    <header className="flex w-full justify-between content-between items-center py-2 px-6 border-b-2 border-black">
+    <header className="flex h-14 w-full items-center justify-between bg-secondary-1 p-4">
+      <div className="h-full">LOGO</div>
       <div>
-        <input className="text-2xl px-2 rounded-md border-2 border-transparent 
-        hover:border-black focus:border-blue-700 focus:outline-none"
-         spellCheck="false" autoComplete="false" type="text" defaultValue="Untitled book"/>
-      </div>
-      <div>
-        <BsPersonFill size={"2rem"} />
+        <img
+          src="./icons/burger_menu.svg"
+          alt="Burger menu"
+          className="h-full cursor-pointer sm:hidden"
+        />
+
+        <div className="hidden flex-row items-center sm:flex">
+          <div className="mr-1 h-6 w-6 rounded-full bg-cover bg-center" style={styles}></div>
```

Comment:
Interesting approach. Why not img tag?

Reply:
Using div for easier stylization and scaling